object_msgs/ObjectPose objectPose
---
bool status
